### ISSUE TITLE
More detail in tei:name needed to be more precise 

### ISSUE THREAD

**Issue by ThomasK81** [brobertson/Lace2#88]
```
<tei:respStmt xmlns:tei="http://www.tei-c.org/ns/1.0">           
<tei:name>surname, forename</tei:name>           \
<tei:resp>proofreader</tei:resp>         
</tei:respStmt>
```

can be misleading. 
I suggest more detail, such as:

```
<tei:respStmt xmlns:tei="http://www.tei-c.org/ns/1.0" ref="[LOD-link that describes the contribution]">
     <tei:resp>[description of the contribution]</tei:resp>
     <tei:persName>
         <tei:forename>forename</tei:forename> 
         <tei:surname>surname</tei:surname>
         <tei:idno type=""></tei:idno> (optional)
     </tei:persName>
 </tei:respStmt>
```

**Comment by brobertson** [comment]
Let's first hack some output to see that this doesn't break TEI-Simple, then implement.

**Comment by brobertson** [comment]
Ugh. It looks like TEISimple doesn't allow this level of specificity:
/Users/brobertson/Downloads/219219004/urn.cts.greekLit.tlg1763.tlg001.1st1K-grc1.xml:2:329: error: element "tei:persName" not allowed anywhere; expected element "name" or "resp"
/Users/brobertson/Downloads/219219004/urn.cts.greekLit.tlg1763.tlg001.1st1K-grc1.xml:2:463: error: element "tei:respStmt" incomplete; missing required element "name"

**Comment by brobertson** [comment]
Resolved by commit 5aadbd1982542d21f930e942ee139773588c14a8
